specification/cognitiveservices/data-plane/VideoAnalysis/videoDescription.tsp

```diff
@@ -0,0 +1,385 @@
+    innererror?: InnerError;
+}
+
+@lroStatus
+@doc("Operation status.")
+union TaskStatusModel {
+    string,
+
+    @doc("The operation has not started yet.")
+    notStarted: "notStarted",
+
+    @doc("The operation is in progress.")
+    running: "running",
+
+    @doc("The operation has failed.")
+    @lroFailed
+    failed: "failed",
+
+    @doc("The operation has succeeded.")
+    @lroSucceeded
+    succeeded: "succeeded",
+
+    @doc("The operation is running and is writting back results.")
+    @lroSucceeded
+    partiallyCompleted: "completed",
+
+    @doc("The operation has failed but wrote partial results back.")
+    partiallyCompletedFailed: "partiallyCompletedFailed",
+
+    @doc("The operation has been canceled.")
+    @lroCanceled
+    canceled: "canceled",
+}
+
+@doc("Preset Configuration")
+union PresetConfigurationModel {
+    string,
+    @doc("Default preset configuration") "default",
+    @doc("Example 1 preset configuration") "example1",
+}
+
+@doc("Authentication model type: None, Key, MI")
+union AuthenticationModelType {
+    string,
+
+    @doc("No authentication")
+    none: "None",
+
+    @doc("Key based authentication")
+    key: "Key",
+
+    @doc("Azure Managed Identity authentication")
+    managedIdentity: "ManagedIdentity",
+}
+
+@doc("Input resource location model kind")
+union InputResourceLocationModelKind {
+    string,
+
+    @doc("Resource Location is publicly available")
+    file: "File",
+
+    @doc("Resource Location is a file on an Azure Storage Account")
+    azureBlobFile: "AzureBlobFile",
+}
+
+@doc("Output resource location model kind")
+union OutputResourceLocationModelKind {
+    string,
+
+    @doc("Resource Location is a container on an Azure Storage Account")
+    azureBlobContainer: "AzureBlobContainer",
+}
+
+@doc("Completion resources model types")
+union CompletionResourceModelType {
+    string,
+
+    @doc("Completion resource based on GPTv APIs")
+    gptv: "Gptv",
+}
+
+@doc("Property Configuration model types")
+union PropertyConfigurationModelType {
+    string,
+
+    @doc("Describe property configuration type")
+    describe: "describe",
+}
+
+@doc("Property items configuration acceptable types")
+union PropertyItemConfigurationModelType {
+    string,
+    @doc("String items") "string",
+}
+
+op VideoAnalysisTaskOperationLongRunningOperation<
+    TParams extends TypeSpec.Reflection.Model,
+    TResponse extends TypeSpec.Reflection.Model
+> is Foundations.Operation<
+    TParams,
+    TypeSpec.Http.AcceptedResponse &
+        Foundations.RetryAfterHeader & {
+            @pollingLocation
+            @header("Operation-Location")
+            operationLocation: ResourceLocation<TResponse>;
+        },
+    JobServiceTraits,
+    VideoAnalysisTaskResponseModel
+>;
+
+@tag("VideoDescription")
+@added(Versions.v2024_05_01_preview)
+interface VideoDescriptions {
+    #suppress "@azure-tools/typespec-azure-core/use-standard-operations" "Doesn't fit standard ops"
```

Comment:
You're going to have to have a strong justification for not using an Azure Core operation template here.